NixSupport/nixosModules/app.nix

```diff
@@ -0,0 +1,25 @@
+# Running an IHP web server + Worker
+{ config, pkgs, modulesPath, lib, ihp, ... }:
+let cfg = config.services.ihp;
+in
+{
+    imports = [
+        ihp.nixosModules.options
+        ihp.nixosModules.services_app
+        ihp.nixosModules.services_worker
+        ihp.nixosModules.services_migrate
+    ];
+
+    # Speed up builds with the IHP binary cache
+    nix.settings.substituters = [ "https://digitallyinduced.cachix.org" ];
+    nix.settings.trusted-public-keys = [ "digitallyinduced.cachix.org-1:y+wQvrnxQ+PdEsCt91rmvv39qRCYzEgGQaldK26hCKE=" ];
+    
+    # Add swap to avoid running out of memory during builds
+    swapDevices = [ { device = "/swapfile"; size = 8192; } ];
```

Comment:
Could it be a custom option in the `flake.nix` for swapfile size? 8 gigs could be a lot for a tiny server.

Reply:
Good point 👍 should be moved to flake.nix